python/tests/integration/connectors/memory/test_qdrant_memory_store.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import numpy as np
```

Comment:
Seeing these failures:


```
FAILED tests/integration/connectors/memory/test_qdrant_memory_store.py::test_qdrant_constructor - AttributeError: 'QdrantMemoryStore' object has no attribute '_client'
FAILED tests/integration/connectors/memory/test_qdrant_memory_store.py::test_create_and_get_collection_async - TypeError: create_collection_async() missing 1 required positional argument: 'vector_size'
FAILED tests/integration/connectors/memory/test_qdrant_memory_store.py::test_get_collections_async - TypeError: create_collection_async() missing 1 required positional argument: 'vector_size'
FAILED tests/integration/connectors/memory/test_qdrant_memory_store.py::test_delete_collection_async - TypeError: create_collection_async() missing 1 required positional argument: 'vector_size'
FAILED tests/integration/connectors/memory/test_qdrant_memory_store.py::test_does_collection_exist_async - TypeError: create_collection_async() missing 1 required positional argument: 'vector_size'
```

Reply:
@lemillermicrosoft Reviewing locally to try to replicate

Reply:
Update corrupt .github files, pushing local updates.